js/ws/index.js

```diff
@@ -10,8 +10,9 @@ var wss = new ws.Server({
 
 if (cluster.isWorker) {
   process.on('message', function(msg) {
+    var msgBuf = Buffer(msg);
     wss.clients.forEach(function each(client) {
-      client.send(msg);
+      client.send(msgBuf);
```

Comment:
You misunderstand me.  I was just saying that this kind of error is not caught by the benchmarking tool.  Your implementation is correct.